ci/scripts/run-skyhook-test

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+# copy test data
+cp -r data /data
+
+# launch cluster
+cd build/
+MON=1 OSD=1 ../src/vstart.sh -d -n -x
```

Comment:
How does this work if you only compile cls_tabular and ceph_test_skyhook_query? Are the binaries in the pulled container?

Reply:
the previous action in the workflow compiles ceph. The ceph folder is bind-mounted, so all binaries are in the workspace